### PR TITLE
rustfmt librand

### PR DESCRIPTION
**Pull request by mseri**
r? @nrc

Re-submission of the closed PR #29054 with the additional rustfmt-zation of the full librand.

### CODE REVIEW

File: src/librand/isaac.rs [hunk 1]
```diff
@@ -113,10 +115,14 @@ impl IsaacRng {
...
-                self.mem[i+4]=e; self.mem[i+5]=f;
-                self.mem[i+6]=g; self.mem[i+7]=h;
+                self.mem[i] = a;
+                self.mem[i + 1] = b;
+                self.mem[i + 2] = c;
+                self.mem[i + 3] = d;
+                self.mem[i + 4] = e;
+                self.mem[i + 5] = f;
+                self.mem[i + 6] = g;
+                self.mem[i + 7] = h;
```
Comment (mseri) [inline src/librand/isaac.rs]: Lines 105-108 in the macro have not been touched, however the same block here has been reformatted. The same happened at line 370. Is this the expected behavior?

Comment (marcusklaas) [inline src/librand/isaac.rs]: Rustfmt can't format macro definitions yet: https://github.com/nrc/rustfmt/issues/8

Comment (mseri) [inline src/librand/isaac.rs]: Should the macro stay in its compact form then, or is it better if it's re-formatted as the rest of the code?

Comment (marcusklaas) [inline src/librand/isaac.rs]: It's to good to be consistent, I think. I'd reformat the macro too, yea.

**Pull request merged**